app/views/shared/_tabnav_account.html.haml

```diff
@@ -3,4 +3,5 @@
   = tab t('shared.tabnav_account.transact'), transactions_account_path(@account), (secondary_tab == 'transactions')
   - if @facility && @account.class.using_statements?
     = tab t('shared.tabnav_account.statement'), account_facility_statement_path(@account, @facility, 'list'), (secondary_tab == 'statements')
+  = tab t('shared.tabnav_account.statement'), account_statements_path(@account), (secondary_tab == 'statements')
```

Comment:
Is this view shared with the admin/facility side? If not, I think we can kill off the previous line. However, we should use `@account.class.using_statements?` (I believe this calls into `Account.config` to only show this tab for Purchase Orders and Credit Cards. Chart Strings don't (for now at least, UIC will want them in an upcoming story, but NU does not).